OSSFE conference event file (#2160)
betterscientificsoftware/bssw.io: Events/2025-03-ossfe.md

# Open Source Software for Fusion Energy (OSSFE) Conference 2025

<!-- deck text start -->

> We invite submissions for the inaugural OSSFE Conference, to be held online on March 18, 2025.
> This event is dedicated to showcasing and advancing open-source software tools for the fusion energy community.

Reviewer: ```suggestion
Join the fusion energy community in showcasing open-source software tools.
```

<!-- deck text ends -->

- **Deadline for abstract submissions**: January 15, 2025
- **Event Date**: March 18, 2025
- **Location**: Online (gather.town)
- **Website**: https://ossfe.github.io
- **Organizers**: Researchers from MIT Plasma Science and Fusion Center, US and International National Labs

#### Contributed by Remi Delaporte-Mathurin

#### Publication date: 9/18/2024

Event Information | Details
:--- | :---
Event Name | [Open Source Software for Fusion Energy (OSSFE) Conference 2025](#)
Deadline for abstract submissions | January 15, 2025
Event Date | March 18, 2025
Website | https://ossfe.github.io

> 

Reviewer: ```suggestion

We invite submissions for the inaugural OSSFE Conference, to be held online on March 18, 2025.
 This event is dedicated to showcasing and advancing open-source software tools for the fusion energy community.
```

OSSFE 2025 is an open and inclusive event aimed at promoting collaboration in open-source software for the fusion energy community.
Topics of interest include (but are not limited to):

- Tritium transport
- Plasma modeling
- Magnet design
- Neutronics
- Best practices in open-source software development for fusion research
- Cross-disciplinary applications of open-source tools in fusion energy

> This event will feature talks, a poster session, and prizes for best presentations.

Reviewer: ```suggestion
This event will feature talks, a poster session, and awards for best presentations.
```

> **Registration is free** to encourage participation from all sectors of the community.

Reviewer: ```suggestion
To encourage participation from all sectors of the community, **registration is free**.
```

Stay updated by visiting the [conference webpage]([#](https://ossfe.github.io)) for further details and updates.

> You can also [subscribe to the newsletter](http://eepurl.com/iYSJJo) to stay up to date with the latest announcements!

Reviewer: @RemDelaporteMathurin I am getting a `502 Bad Gateway` for this link to the newsletter.

Reviewer: Well, let me clarify...I get that when previewing the page from the bssw.io preview site. I don't get that error when I follow that link directly from viewing this file on GitHub.com. Thoughts?

Author: I also can access it when I follow the link directly. Could it be a https vs http issue?

Reviewer: Well, I can follow the link from preview site now too. I wonder if that is because I first successfully followed it from github.com. I dunno. Its working now. So, moving on.
